python/ray/_private/runtime_env/context.py

```diff
@@ -34,10 +34,13 @@ def serialize(self) -> str:
     def deserialize(json_string):
         return RuntimeEnvContext(**json.loads(json_string))
 
-    def exec_worker(self, passthrough_args: List[str]):
+    def exec_worker(self, passthrough_args: List[str], language: str):
```

Comment:
Can we make language an ENUM please?

Reply:
Good suggestion. I have used the language enum from protobuf. https://github.com/ray-project/ray/pull/19060/commits/3341139cef661d959606071ac8b9606697bb4156